### DISCUSSION TITLE
use `eq` [Jul 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2619] [Discord Threads]
> 💁 **o1lab**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Paweł**  
#### it does not work    
#### `{   
####   "msg": "select \"sheet1\".\"created_at\" as \"CreatedAt\", \"sheet1\".\"updated_at\" as \"UpdatedAt\", \"sheet1\".\"json\" as \"json\", \"sheet1\".\"status\" as \"status\", \"sheet1\".\"receipt_id\" as \"receipt_id\", \"sheet1\".\"source_system\" as \"source_system\" from \"sheet1\" where ((((\"receipt_id\" in (select \"sheet1_id\" from \"etsy_payment\" where \"payment_id\" = 'null'))))) order by \"created_at\" asc limit 1 - invalid input syntax for integer: \"null\""   
#### }`  
#  
> 🙋‍♂️ **wkw**  
#### <@708648416145702952>  please create an issue. seems it s not expected. will take a look  
#  
> 🙋‍♂️ **Paweł**  
#### https://github.com/nocodb/nocodb/issues/2516  
#  
> 🙋‍♂️ **wkw**  
#### thanks. will investigate  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/992007779973877820)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
